workers/nightly_build_github_worker.rb

```diff
@@ -0,0 +1,113 @@
+require_relative "worker_base"
+require_relative "../services/build_service"
+require_relative "../shared/models/job_trigger"
+require_relative "../shared/models/provider_credential"
+require_relative "../shared/logging_module"
+require_relative "../services/test_runner_service"
+require_relative "../services/code_hosting/git_hub_service"
+
+module FastlaneCI
+  # Responsible for starting off builds nightly
+  # TODO: determine what timezone to use
+  class NightlyBuildGithubWorker < WorkerBase
```

Comment:
This is branched from CheckForNewCommitGithubWorker
http://www.mergely.com/Tr0JT2n4/